docs/deprecated.md

```diff
@@ -0,0 +1,3 @@
+⚠️ cozy-client-js is a depreacted librairy. We recommend you to use our new client : [Cozy Client](https://docs.cozy.io/en/cozy-client/getting-started/)
+
+If you really need to see the documention of cozy-client-js, see it on [github](https://github.com/cozy/cozy-client-js/blob/master/docs/README.md)
```

Comment:
Maybe we can set up again github pages for cozy-client-js

Reply:
IMHO, we should not spend more time on cozy-client-js and the github documentation is sufficient. Do you see any advantage of github pages over normal github for documentation (apart from being nicer looking) ?

Reply:
It looks nicer, and it is just two clicks to setup.